src/System.Runtime/tests/System/Reflection/AssemblyTests.cs

```diff
@@ -431,13 +440,14 @@ public void UnsafeLoadFrom_SamePath_ReturnsEqualAssemblies()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The implementation of LoadFrom(string, byte[], AssemblyHashAlgorithm is not supported in .NET Core.")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, "The implementation of LoadFrom(string, byte[], AssemblyHashAlgorithm is not supported in .NET Core.")]
```

Comment:
What is our story in general when an API works on CoreCLR but not in AOT -- this means it works for F5 but not in release. Do we help customers find out when they write such code?